xml/src/main/java/com/baeldung/sax/SaxParserMain.java

```diff
@@ -0,0 +1,161 @@
+package com.baeldung.sax;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class SaxParserMain {
+    public static void main(String[] args) throws ParserConfigurationException, SAXException, IOException {
+        SAXParserFactory factory = SAXParserFactory.newInstance();
+        SAXParser saxParser = factory.newSAXParser();
+
+        BaeldungHandler baeldungHandler = new BaeldungHandler();
+        saxParser.parse("xml\\src\\main\\resources\\sax\\baeldung.xml", baeldungHandler);

```

Comment:
can we use forward slash here?

Reply:
Yes, I corrected it in the test but not here. Good point